pkg/phlaredb/metrics.go

```diff
@@ -201,8 +207,13 @@ func newBlocksMetrics(reg prometheus.Registerer) *blocksMetrics {
 			Name: "pyroscopedb_block_opening_duration",
 			Help: "Latency of opening a block in seconds",
 		}),
+		blockOpened: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "pyroscopedb_block_opened",
```

Comment:
The name is not very clear

```suggestion
			Name: "pyroscopedb_blocks_open",
```

or 

```suggestion
			Name: "pyroscopedb_blocks_currently_open",
```